tools/SecurityToolsScripts/RunBinSkim.ps1

```diff
@@ -0,0 +1,10 @@
+$repoRoot = "$PSScriptRoot/../.."
+$binSkim = (Get-Item -Path "$repoRoot/tools/SecurityTools/Microsoft.CodeAnalysis.BinSkim*/tools/*/win-x64/BinSkim.exe" | Select-Object -First 1).FullName
+$dllBaseFolder = "$repoRoot/artifacts/Debug"
+$suppressions = @('Microsoft.Azure.DataLake.Store.dll','Microsoft.WindowsAzure.Storage.DataMovement.PowerShell.dll')
```

Comment:
Do these suppressions come from failures or from previous suppression?

Reply:
These are new failures. It seems these are signed with Sha1+Sha2. So, to avoid these suppressions, we would need resigned DLLs. I did, however, want to have a suppression strategy for BinSkim since it natively doesn't allow it.